services/media/service/.env.template

```diff
@@ -13,6 +13,10 @@ DATABASE_LOGIN=media_service_login
 DATABASE_LOGIN_PASSWORD=media_service_login_pwd
 DATABASE_GQL_ROLE=media_service_gql_role
 
+# NOTE: If these values change the old database replication slots must be explicitly dropped.
+TRANSACTIONAL_OUTBOX_REPLICATION_SLOT=ott_media_service_transactional_outbox_slot
```

Comment:
Please also adjust the mosaic-hosting-deployment-manifest.yaml. Check
`feature/localization` branch for example, as we added slot name as empty and
advice using comment to generate a unique name using script (already mentioned
in previous comment)